### PR TITLE
Refresch a bit the 'packages' CI/CD with latest version of actions/distrib

### PR DESCRIPTION
**Pull request by kelson42**
* Refresh a bit the configuration
* Add ubuntu-moble to CI/CD
* Add the latest Debian to the CI
* Remove old `zimwriter` deb package

### CODE REVIEW

**Comment by kelson42** [comment]
Maybe @tusharpm you can have a look to that PR and you have a few feedbacks?

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -46,6 +50,39 @@ jobs:
         id: build-debian-unstable
         with:
           args: --no-sign
+          ppa: ${{ steps.ppa.outputs.ppa }}
```
Comment (tusharpm) [inline .github/workflows/package.yml:53]: Debian builds should not be referencing Ubuntu PPAs. The packages are published independently at https://packages.debian.org/search?keywords=libzim.
Requesting @legoktm to upload libzim9 to the other Debian versions (only available on experimental so far).

File: debian/zim-tools.install [hunk 2]
```diff
@@ -1 +1 @@
-usr/bin/
+usr/bin/*
```
Comment (tusharpm) [inline debian/zim-tools.install:1]: Ubuntu packaging step is failing due to this path expression. Let's try running locally to use the correct expression here.

File: .github/workflows/package.yml (hunk 1 above)
Comment (kelson42) [reply .github/workflows/package.yml]: > Debian builds should not be referencing Ubuntu PPAs.

Fixed

File: debian/zim-tools.install (hunk 2 above)
Comment (kelson42) [reply debian/zim-tools.install]: I have fixed this regression I have added this regression

**Comment by kelson42** [comment]
@tusharpm Do you know why we have [this error](https://github.com/openzim/zim-tools/actions/runs/10538605357/job/29201288215?pr=424#step:18:459) on Ubuntu packages? For Debian, I guess it fails because - like you said - packages are not updated upstream in Debian repositories (ping @legoktm)

**Comment by kelson42** [comment]
@tusharpm So, finally it seems the `zim-tools.install` file was really not necessary. I have removed it and now the Ubuntu deb files seem OK. But the Debian deb file still fail to build because libzim `9.2` has not been upload or not available at all. @legoktm or @tusharpm Are you able to help with this last problem?

**Comment by tusharpm** [comment]
> @tusharpm So, finally it seems the `zim-tools.install` file was really not necessary. I have removed it and now the Ubuntu deb files seem OK.

Nice catch! I don't know whether this impacts the output deb file in any way. I'll try spinning up a docker container and test them from the launchpad PPA.

> But the Debian deb file still fail to build because libzim `9.2` has not been upload or not available at all. @legoktm or @tusharpm Are you able to help with this last problem?

Sadly, I'm not a [Debian package maintainer](https://salsa.debian.org/debian/libzim). I'll defer to @legoktm for comment on updating the Debian pacakages.

**Comment by kelson42** [comment]
> > But the Debian deb file still fail to build because libzim `9.2` has not been upload or not available at all. @legoktm or @tusharpm Are you able to help with this last problem?
> 
> Sadly, I'm not a [Debian package maintainer](https://salsa.debian.org/debian/libzim). I'll defer to @legoktm for comment on updating the Debian pacakages.
> 
> Meanwhile, I suggest to continue using the previous (available) version of libzim if it doesn't have a major drawback/security vulnerability.

This version of libzim relies on libzim 9.2, there is nothing I believe I can do about that. For the rest I kind of believe that @legoktm would be kind of consider to have a deputy, so don't be shy and ask him if you want and believe you have enough skills. For Kiwix this would be great, because it is not rare that @legoktm lack of time.

**Pull request merged**